cypress/support/pages/group/pageGroups.js

```diff
@@ -0,0 +1,156 @@
+'use strict'
```

Comment:
1) Here as well the folder name would be 'class_management' instead of 'group'
2) Then the page object filename would be -> pageClasses.js